virttest/storage.py

```diff
@@ -441,6 +441,29 @@ def get_backup_set(filename, backup_dir, action, good):
                 continue
             backup_func(src, dst)
 
+    def rm_backup_image(self, only_backup=True):
+        """
+        Remove backup images, if requested, as well original image
+
+        :param only_backup: flag to decide removal of only backup images
+        or removal of original image along with backup images.
+        """
+        rm_image = []
+        backup_dir = utils_misc.get_path(self.root_dir,
+                                         self.params.get("backup_dir", ""))
+        rm_image.append(os.path.join(
+            backup_dir, "%s.backup" % self.image_filename))
+        if not only_backup:
+            logging.debug("Removing original as well as backups of image %s",
+                          self.image_filename)
+            rm_image.append(self.image_filename)
```

Comment:
I think we should not consider removing the original one via this method,

Reply:
Ok then for deleting original image we should use qemu_storage remove method. Made changes here only to remove backup image.